test/e2e/framework/framework.go

```diff
@@ -24,6 +25,7 @@ func setup() error {
 		defaultKubeConfig = homedir + "/.kube/config"
 	}
 	config := flag.String("kubeconfig", defaultKubeConfig, "kubeconfig path, defaults to $HOME/.kube/config")
+	extRepo := flag.String("external_repo", "", "external repo to push to, defaults to none (builds image to local docker repo)")
```

Comment:
Why not have this as the complete image name instead of just the repository name without the tag:
```Go
imageName := flag.String("image", "", "operator image name <repository>:<tag> used to push the image, defaults to none (builds image to local docker repo)")
```
And then no need to append v0.0.1 to the image name later.